Adding my advice page to ongoing advice

charlottegray17 opened the pull request.

redrambles commented:
Hey @charlottegray17 !

This is great - we just have two little things we need you to fix up before we can accept your pull request:

1) Unfortunately we cannot accept images as part of student pull requests, as the repository would soon get too large - so we'll need you to delete `img/topic_area_bg.jpg`. You are free to link to a version of the image that is hosted somewhere on the internet, though, of course! :) 

2) We'll need your advice html page inside the `advice` directory. Right now it is in the root of the project.

Once you have made these changes, please add with `git add -A` - then commit as per usual and push up your advice branch once more, which will automatically update this pull request.

Then we should be all set to merge in your code. Let us know if you have any questions!

Cheers,
Ann :)

charlottegray17 commented:
Hi Ann,

Thanks so much for your mail. I knew I must have done something wrong!

I have addressed your points, which made complete sense, thank you.

I updated my master, re-pushed my repo and created a new pull request - but now re-reading your mail, I see I wasn't supposed to do that (oh dear, sorry!).

Hopefully this new request doesn't cause too much trouble and can merge in with the others. 🙂

All the best from troublesome London student,

Charlotte
________________________________
From: Ann Cascarano <notifications@github.com>
Sent: Tuesday, March 31, 2020 20:28
To: skillcrush/105-practice <105-practice@noreply.github.com>
Cc: Charlotte Gray <charlotte.gray17@hotmail.co.uk>; Mention <mention@noreply.github.com>
Subject: Re: [skillcrush/105-practice] Adding my advice page to ongoing advice (#1021)


Hey @charlottegray17<https://github.com/charlottegray17> !

This is great - we just have two little things we need you to fix up before we can accept your pull request:

  1.  Unfortunately we cannot accept images as part of student pull requests, as the repository would soon get too large - so we'll need you to delete img/topic_area_bg.jpg. You are free to link to a version of the image that is hosted somewhere on the internet, though, of course! :)

  2.  We'll need your advice html page inside the advice directory. Right now it is in the root of the project.

Once you have made these changes, please add with git add -A - then commit as per usual and push up your advice branch once more, which will automatically update this pull request.

Then we should be all set to merge in your code. Let us know if you have any questions!

Cheers,
Ann :)

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/skillcrush/105-practice/pull/1021#issuecomment-606796233>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AI36JVMBJVJOKFS3NAOYEKDRKIY55ANCNFSM4LV5AHGA>.
